config/rbac/role.yaml

```diff
@@ -18,6 +18,14 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - list
+  - delete
+  - create
```

Comment:
While that works, MCAD should theoretically also be able to update these resources, so I'd suggest to add the following verbs for consistency:

```
- patch
- update
- watch
```

Reply:
Thanks @astefanutti  :) I have applied the update in aa18a415df6598f91d20f785dc64a967c140a411